Common/Source/Dialogs/dlgRadioSettings.cpp

```diff
@@ -448,25 +458,33 @@ static void OnRadioActiveAutoClicked(WndButton* pWnd){
 		if ( ValidWayPoint(BestAlternate))
 		{
 			double fFreq = StrToDouble(WayPointList[BestAlternate].Freq,NULL);
-			if(ValidFrequency(fFreq))
-				devPutFreqActive(fFreq, WayPointList[BestAlternate].Name);
+
+			if(ValidFrequency(fFreq)) {
+				RadioPara.ActiveFrequency = fFreq;
+				CopyTruncateString(RadioPara.ActiveName, NAME_SIZE, WayPointList[BestAlternate].Name);
+				devPutFreqActive(	RadioPara.ActiveFrequency , RadioPara.ActiveName);
```

Comment:
this 3 line are same as 
    devPutFreqActive(fFreq, WayPointList[BestAlternate].Name);